dotnet/src/webdriver/Proxy.cs

```diff
@@ -239,8 +242,12 @@ public bool IsAutoDetect
     /// <summary>
     /// Gets or sets the value of the proxy for the FTP protocol.
     /// </summary>
+    /// <remarks>
+    /// FTP proxy support is deprecated and will be removed in a future version.
+    /// </remarks>
     [JsonPropertyName("ftpProxy")]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+    [Obsolete("FTP proxy support is deprecated and will be removed in a future version.")]
```

Comment:
```suggestion
    [Obsolete("FTP proxy support is deprecated and will be removed in the 4.37 version.")]
```

This follows standard Selenium's deprecation policy.

Only this line of code is expected in this PR. All other changes are unnecessary, because it is not user-facing changes. Even if it is just a note for us. When we remove property, compiler highlights what we should remove.

Reply:
no need to store comments like 
```
            // TODO: Remove ftpProxy in future version and remove deprecation warning
            // https://github.com/SeleniumHQ/selenium/issues/15905
```
am i right ?

Reply:
thanks for advice !